src/controllers/admin-controller/multicast.controller.ts

```diff
@@ -73,8 +73,7 @@ export class MulticastController {
                 ActionType.CREATE,
                 Multicast.name,
                 req.user.userId,
-                multicast.lorawanMulticastDefinition.chirpstackGroupId,
-                multicast.groupName
+                multicast.id
```

Comment:
Log both the id and `groupName` of `multicast` so it's easier to identify the object in the log